tensorflow/python/keras/_impl/keras/estimator.py

```diff
@@ -274,7 +287,7 @@ def _clone_and_build_model(mode,
                                         is_input=False)
   else:
     target_tensors = [
-        _cast_tensor_to_floatx(
+        _convert_tensor(
             sparse_tensor_lib.convert_to_tensor_or_sparse_tensor(labels))
```

Comment:
Thanks @Dref360! nit, we can remove sparse_tensor_lib.convert_to_tensor_or_sparse_tensor here?